opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/metric_reader_storage.py

```diff
@@ -115,12 +116,17 @@ def collect(self) -> Iterable[Metric]:
         # effect is that end times can be slightly skewed among the metric
         # streams produced by the SDK, but we still align the output timestamps
         # for a single instrument.
+
+        collection_start = _time_ns()
```

Comment:
nit I think `collection_start_nanos` throughout would be much clearer for reading the code (since it's just int)

Reply:
Fixed